src/commands.ts

```diff
@@ -369,6 +372,7 @@ async function openWorkspace(
   workspaceName: string,
   workspaceAgent: string | undefined,
   folderPath: string | undefined,
+  openRecent: boolean | undefined,
```

Comment:
Any particular reason for not using optional parameters via `?:`?

Reply:
Just to make sure it is specified.  I find sometimes when I add new arguments, if I make it optional with `?:` I can miss areas where I should have added it.  But doing it this way forces me to check every function call since they will error.

Once that is done, I might change it to `?:` after some consideration, but in this case since in all our calls to this function we have a value to pass, I did not bother.

Reply:
Yeah, that makes sense. I think that I've favored the optional syntax for my own convenience up 'til now, but I can see how forcing all the parameters to be explicit would be handy